.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
```

Comment:
I would recommend enabling manual deployment approval (the same as we do when we deploy to production on the backend)